.github/workflows/verify-merge.yml

```diff
@@ -0,0 +1,14 @@
+name: ci/gh-actions/verify-merge
+
+on: [push, pull_request]
+
+jobs:
+  verify-merge:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: recursive
```

Comment:
no submodules

Reply:
Just wanted to code it a bit "defensively". Removed:

https://github.com/mj-xmr/gitian.sigs/actions/runs/737468964